.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,7 +22,7 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Run `make reviewable test` to ensure this PR is ready for review.
```

Comment:
Please remove this change as we require that tests pass for changes to be approved.

Reply:
Yes, sure! But, one question- if I run `make test` I get the response: `make: *** No rule to make target `test'.  Stop.` and I cannot find a reference in the Makefile defining the cmd.

Reply:
Make sure you initialize and pull the submodules! I believe it's in the submod for Crossplane's common build utilities.